auth/authenticate.js

```diff
@@ -0,0 +1,43 @@
+const jwt = require('jsonwebtoken');
+
+const jwtKey =
+  process.env.JWT_SECRET ||
+  'add a .env file to root of project with the JWT_SECRET variable';
+
+// quickly see what this file exports
+module.exports = {
+  authenticate,
+  generateToken
+};
+
+function generateToken(user) {
+  const payload = {
+      username: user.username,
```

Comment:
Then you can just do 
```suggestion
      username,
```
Works both the same 👍

Reply:
I like it thank you sir 👍